sym_api_client_python/services/abstract_datafeed_event_service.py

```diff
@@ -25,10 +25,11 @@ def __init__(self, sym_bot_client, error_timeout_sec=None, maximum_timeout_sec=N
         self.registered_triggers = []
 
         self.bot_client = sym_bot_client
+        self.config = sym_bot_client.get_sym_config()
         self.datafeed_client = self.bot_client.get_datafeed_client()
+        self.datafeed_id_repository = OnDiskDatafeedIdRepository(self.config.get_datafeed_id_folder_path())
```

Comment:
as it is specific to v1 only is it a logic we could push down to the subclass?

Reply:
Indeed but it would require to duplicate this logic in `AsyncDataFeedEventService` and in `DataFeedEventServiceV1`.